kudo-e2e-test.yaml.tmpl

```diff
@@ -3,7 +3,7 @@ kind: TestSuite
 testDirs:
 - ./test/e2e
 commands:
-- command: ./bin/kubectl-kudo init --unsafe-self-signed-webhook-ca --kudo-image kudobuilder/controller:%version% --kudo-image-pull-policy Never
+- command: ./bin/kubectl-kudo init --wait --unsafe-self-signed-webhook-ca --kudo-image kudobuilder/controller:%version% --kudo-image-pull-policy Never
```

Comment:
hmm is this some kind of bad merge? Because we did this in a different PR 🤔 or you added it heree as well before it's on the master?

Reply:
I've added it here as well before it was on master. Shouldn't show up anymore